pyproject.toml

```
@@ -1,3 +1,47 @@
+[project]
+name = "spidev"
+description = "Python bindings for Linux SPI access through spidev"
+dynamic = ["version", "readme"]
+
+authors = [
+    { name = "Volker Thoms", email = "unconnected@gmx.de" },
+]
+maintainers = [
+    { name = "Stephen Caudle", email = "scaudle@doceme.com" },
+]
+license = { text = "MIT", file = "LICENSE" }
```

This seems to be deprecated and replaced by a new format in `PEP 639` introduced in `setuptools` 77.0.3. See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-and-license-files for details.

```suggestion
license = "MIT"
license-files = ["LICENSE"]
```